include/bluetooth/hci.h

```diff
@@ -1736,6 +1736,10 @@ struct bt_hci_rp_le_reject_cis {
 	uint16_t handle;
 } __packed;
 
+#define BT_HCI_LE_BIG_PHY_1M                    BIT(0)
+#define BT_HCI_LE_BIG_PHY_2M                    BIT(1)
+#define BT_HCI_LE_BIG_PHY_CODED                 BIT(2)
+
```

Comment:
Do we need these when we already have the following? 

```
/** LE PHY types */
enum {
	/** Convenience macro for when no PHY is set. */
	BT_GAP_LE_PHY_NONE                    = 0,
	/** LE 1M PHY */
	BT_GAP_LE_PHY_1M                      = BIT(0),
	 /** LE 2M PHY */
	BT_GAP_LE_PHY_2M                      = BIT(1),
	/** LE Coded PHY */
	BT_GAP_LE_PHY_CODED                   = BIT(2),
};
```

Reply:
Hmm, I guess since we also have 

```
#define BT_HCI_LE_EXT_SCAN_PHY_1M               BIT(0)
#define BT_HCI_LE_EXT_SCAN_PHY_2M               BIT(1)
#define BT_HCI_LE_EXT_SCAN_PHY_CODED            BIT(2)
```

That might be the norm to add for each command?

Reply:
Will close this PR, will use the BT_GAP_LE_PHY for BIG APIs, (Please add to BIG API documentation to pass these enum values  as parameters)